cfgs/robustness_analysis/pretraining_experiment/pretrain_0.toml

```diff
@@ -1,23 +1,19 @@
-name = "polyhedral-noise-seed-37"
+name = "pretrain-0"
 cuda_devices = "0"
 plotting_points = 2500
+resolution = "10%"
```

Comment:
0.1 as string with % makes me flinch :)

Reply:
Agreed :)

The reasoning behind this is that this maps to a fixed set of file names in the end, and floating point numbers aren't that good when it's about equality comparisons.

Reply:
I will try to erase any memory of this from my mind :)